src/bionics.cpp

```diff
@@ -1656,18 +1656,26 @@ void Character::process_bionic( int b )
                        static_cast<std::string>( bio_hydraulics ) );
     } else if( bio.id == bio_nanobots ) {
         if( get_power_level() >= 40_J ) {
-            std::forward_list<int> bleeding_bp_parts;
+            std::vector<body_part> bleeding_bp_parts;
```

Comment:
```suggestion
            std::vector<bodypart_id> bleeding_bp_parts;
```
I'm trying to replace `body_part` with `bodypart_id` (see #39068) to make body parts jsonable, it would be helpful if could use the id instead of the enum.